src/main/webapp/index.html

```diff
@@ -1,19 +1,32 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>Sign In</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
+    <!-- <img src="Images/PizzaBackground.png" width="50%" height="100%" ; /> -->
```

Comment:
this shouldnt be an image. should be built with html css. you can download the image from figma. font doesnt matter